mypy_django_plugin/transformers/models.py

```diff
@@ -273,6 +277,12 @@ def run(self) -> None:
             return None
         meta_node.fallback_to_any = True
 
+        typed_model_meta_info = self.lookup_typeinfo(fullnames.TYPED_MODEL_META_FULLNAME)
+        if typed_model_meta_info and not meta_node.has_base(fullnames.TYPED_MODEL_META_FULLNAME):
+            # Insert TypedModelMeta just before `object` to leverage mypy's class-body semantic analysis.
+            meta_node.mro.insert(-1, typed_model_meta_info)
```

Comment:
Do we need to add this base class to `ClassDef` as well?

Reply:
I don't think because we could endup with multiple `TypedModelMeta` in the mro for multiple inheritance cases.